app/models/book.rb

```diff
@@ -88,6 +88,75 @@ class Book < ActiveRecord::Base
   has_and_belongs_to_many :recommendations
 
   CSV_COLUMNS = ["Book Name", "ASIN"]
+  LEVELS_DICT = {
```

Comment:
hmm not sure if there is a better way of doing this... @wonjun @SamLau95 thoughts?